soc/riscv/riscv-ite/it8xxx2/Kconfig.defconfig.series

```diff
@@ -13,6 +13,9 @@ config ITE_IT8XXX2_INTC
 	select FLASH_HAS_DRIVER_ENABLED
 	select HAS_FLASH_LOAD_OFFSET
 
+config ITE_IT8XXX2_WUC
+	default y
+
```

Comment:
Is this necessary?

Wonẗ it be default y due to dt_nodelabel_has_compat,wuc,it8xxx2-wuc being true ?

Reply:
Yes, it's necessary, or it shows:  undefined reference wuc driver functions when build.

Reply:
This should be removed as it is DT that should be determining the default value.